Source/Dafny/Compilers/Compiler-Csharp.cs

```diff
@@ -2381,7 +2399,7 @@ protected class ClassWriter : IClassWriter {
 
       var w = new ConcreteSyntaxTree();
       if (to.IsRefType) {
-        wr.Format($"({TypeName(to, wr, tok)})({w})");
+        wr.Format($"(({TypeName(to, wr, tok)})({w}))");
```

Comment:
Is this a bugfix?

Reply:
It was, I removed it from the branch (I shouldn't have pushed it)